modules/gitops-repo-server-properties.adoc

```diff
@@ -23,6 +23,8 @@ The following properties are available for configuring the Repo server component
 |`execTimeout` | `180` |Execution timeout in seconds for rendering tools (for example Helm or Kustomize).
 |`env` | `__<empty>__` |Environment to set for the repository server workloads.
 |`replicas` | `__<empty>__` |The number of replicas for the Argo CD Repo server. Must be greater than or equal to `0`.
+|`volumes` | `__<empty>__` |Configures additional volumes used for the repo server deployment. This field is optional.
+|`volumeMounts` | `__<empty>__` |Configures additional volume mounts used for the repo server deployment. This field is optional.
```

Comment:
Similarly, noticing "Repo server" vs "repo server" here (seems split across our openshift-docs repo as well). Only a nit really, but would be better if there was consistency.

Reply:
Fixed.